mantidproject/mantid
# Improve Reflectometry Help Facilities

- Adds the usual `?` button to the reflectometry interface which links to it's corresponding documentation.
- Makes line edit hints and tooltips readable on Linux distros.
- Sets tooltips to corresponding property documentation on the Reflectometry settings GUI.
- Adds more tooltips to the Runs and Event tabs, excluding the `Save ASCII` tab for now due to imminent changes.

**To test:**
1. Open the reflectometry GUI and ensure that the `?` button in the lower left corner opens the help browser as expected.
2. On Linux 
   1. Go to the settings tab and click in the `Stitch1DMany` box, begin typing and ensure that the suggestions are readable.
   2. Returning to the runs tab, press the `->?` button on the toolbox to enable 'What's this' mode.
   3. Click on one of the other tools in the toolbox and ensure that the help is also readable.
3. Load the sample table `INTER_NR_test2.tbl` from the ISIS sample data by clicking `Reflectometry->Import TBL` and selecting the file. Process all the runs before switching to the settings tab and editing a setting. Ensure that the rows are no longer highlighted in green.



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
